deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -291,8 +291,7 @@ spec:
             selector:
               matchLabels:
                 app: che-operator
-            strategy:
-              type: Recreate
+            strategy: {}
```

Comment:
Maybe instead of default one better to explicitly set which we expect, like Rolling?

Reply:
I've decided just to rollback. 
Once I configure readiness and liveness probe I will set the strategy explicitly.